test/connectionProfile.test.ts

```diff
@@ -14,9 +15,10 @@ import { ConnectionUI } from '../src/views/connectionUI';
 import { ConnectionStore } from '../src/models/connectionStore';
 import ConnectionManager from '../src/controllers/connectionManager';
 import VscodeWrapper from '../src/controllers/vscodeWrapper';
-
+import Constants = require('../src/constants/constants');
```

Comment:
same here.  I think in typescript we generally prefer an `import..from` variation to calling require directly.  Is there a reason to use require directly here?

Reply:
I think I had just copied those imports from elsewhere, I can change them to the `import from` form